assembl/static2/js/app/components/debate/thread/editPostForm.jsx

```diff
@@ -93,7 +103,22 @@ class EditPostForm extends React.PureComponent<void, EditPostFormProps, EditPost
     }
   };
 
+  deleteAttachment = (documentId) => {
+    this.setState(
+      {
+        deletingAttachment: true
+      },
+      () => {
+        const newBody = attachmentsPlugin.removeAttachment(this.state.body, documentId);
```

Comment:
So, by my understanding, documents will be saved, however, `postAttachments` will not be created. If the attachment is cancelled before saving, then the document is stored in the DB still? If so, seems like a quick way to bloat the DB.

Reply:
yes, I talked about it during the stand-ups. It is definitely something that we have to deal with quickly